beacontools/packet_types/eddystone.py

```diff
@@ -98,8 +98,8 @@ class EddystoneTLMFrame(object):
 
     def __init__(self, data):
         self._voltage = data['voltage']
-        self._temperature = data['temperature']
-        self._temperature_fixed_point = data['temperature'] / float(256)
```

Comment:
What is the reason behind this change?

Reply:
Aaah I see. I parsed the temperature wrong. It must big-endian and then divided by 256, which results in the exact same value if you have no decimals like the beacons I am using🤦‍♂️

Reply:
You can remove `_temperature_fixed_point` attribute completely then.